doc/io/ingestr/index.md

```diff
@@ -44,47 +82,140 @@ Amazon S3, Google Cloud Storage
 Airtable, Asana, GitHub, Google Ads, Google Analytics, Google Sheets, HubSpot,
 Notion, Personio, Salesforce, Slack, Stripe, Zendesk, etc.
 
-## What's inside
+## Configure
+
+For hands-on examples about the configuration parameters enumerated here,
+see [usage section](#usage) below.
+
+:::{rubric} Data source and destination
+:::
+[ingestr] uses four parameters to address source and destination, while
+[`ctk load table`] uses just two, by embedding the source and destination
+table names into the address URLs themselves, using the `table` query
+parameter.
+
+:::{rubric} Batch size
+:::
+Because the underlying framework uses [dlt], you will configure parameters like
+batch size in your `.dlt/config.toml`.
+```toml
+[data_writer]
+buffer_max_items=1_000
+file_max_items=100_000
+file_max_bytes=50_000
+```
 
-The CrateDB destination adapter for ingestr is implemented using [dlt].
+:::{rubric} Incremental loading
+:::
+ingestr supports [incremental loading], which means you can choose to append,
+merge or delete+insert data into the destination table using different
+strategies. Incremental loading allows you to ingest only the new rows from
+the source table into the destination table, which means that you don't have
+to load the entire table every time you run the data migration procedure.
+
+:::{rubric} Credentials
+:::
+**Source:** Data pipeline source elements use their specific way for
+configuring access credentials, using individual parameters.
 
-<iframe width="240" src="https://www.youtube-nocookie.com/embed/iNxRemknAdQ?si=J0w5yG56Ld4fIXfm" title="dlt (data load tool) - Python data extraction / loading tool" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+**Destination:** CrateDB as the pipeline sink element uses the same way to
+specify credentials across the board within the `--cluster-url` CLI option.
+Please note you must specify a password. If your account doesn't use a password,
+use an arbitrary string like `na`.
+
+(ingestr-custom-queries)=
+:::{rubric} Custom queries
+:::
+ingestr provides [custom queries for SQL sources], by using the `query:` prefix
+to the source table name. The feature can be used for partial table loading
+to limit the import to use just a subset of the source columns, or for general
+data filtering and aggregation purposes. Example:
+```shell
+ctk load table \
+    "crate://crate:na@localhost:4200/?table=query:SELECT * FROM sys.summits WHERE height > 4000" \
+    --cluster-url="crate://crate:na@localhost:4200/testdrive/cratedb_summits"
+```
```

Comment:
Note: ingestr's [custom queries](https://bruin-data.github.io/ingestr/supported-sources/custom_queries.html) feature can resemble a little fragment of a data shoveling machinery that might relate to requirements of emulated materialized views or table rollups.